### PR TITLE
Updating json file

### PR DESCRIPTION
**Pull request by BKorenek**
Updating package json to reflect the correct dev dependencies version and description.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "Amy Troppe <amy@medology.com>",
     "Donald Tyler <donald@medology.com>",
     "Joel Padron <joel@medology.com>",
-    "Josh Medeski <josh@medology.com>"
+    "Josh Medeski <josh@medology.com>",
+    "Brandon Korenek <brandon@healthlabs.com>"
```
Comment (Chekote) [inline package.json]: Who is this "HealthLabs" guy?

File: package.json [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
+    "jquery": "^2.2.3"
   },
   "devDependencies": {
     "resolve-url-loader": "^3.1.0",
     "sass": "^1.23.1",
     "sass-loader": "^8.0.0",
-    "vue-template-compiler": "^2.6.10"
+    "vue-template-compiler": "^2.6.10",
+    "laravel-mix": "^5.0.0"
+
```
Comment (juanmrad) [inline package.json]: remove this extra empty line.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "findalab",
-  "version": "2.8.0",
-  "description": "Find a Lab jQuery plugin.",
+  "version": "3.0.0",
```
Comment (juanmrad) [inline package.json]: version will be 3.0.1 since this will require a new release.

File: package.json (hunk 1 above)
Comment (BKorenek) [reply package.json]: He's a legendary developer that roams the halls at night, doing dirty things at peoples desks and stealing headphones

**Pull request merged**